AppImageBuilder.yml

```diff
@@ -0,0 +1,68 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
+version: 1
+
+script: |
+  mkdir -p build/AppDir build/AppDir/usr/share/icons/hicolor/scalable/apps
+  cp _kosmorro/assets/svg/kosmorro-icon.svg build/AppDir/usr/share/icons/hicolor/scalable/apps/kosmorro.svg
+  pip3 install --isolated --root="./build/AppDir" ./dist/kosmorro-*.whl
+
+AppDir:
+  path: build/AppDir
+  app_info:
+    id: space.kosmorro.cli
+    name: kosmorro
+    icon: kosmorro
+    version: latest
+    exec: usr/bin/python3
+    exec_args: $APPDIR/usr/local/bin/kosmorro $@
+
+  apt:
+    arch: amd64
+    allow_unauthenticated: true
+    sources:
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal main restricted
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal universe restricted
+    include:
+      - python3
+      - python3-certifi
+      - python3-dateutil
+      - texlive
```

Comment:
`pdflatex` binary does not seem to be found by Kosmorro, even though it's in the `$PATH`.

Reply:
Looks like LaTeX really needs to be installed system-wide. Let's remove it from AppImage for stability.